test/extensions/filters/http/aws_lambda/aws_lambda_filter_test.cc

```diff
@@ -0,0 +1,161 @@
+#include "envoy/config/core/v3/base.pb.h"
+
+#include "extensions/filters/http/aws_lambda/aws_lambda_filter.h"
+#include "extensions/filters/http/well_known_names.h"
+
+#include "test/mocks/http/mocks.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace AwsLambdaFilter {
+
+namespace {
+
+using ::testing::Invoke;
+using ::testing::Return;
+using ::testing::ReturnRef;
+
+class MockSigner : public Common::Aws::Signer {
```

Comment:
isnt there already a MockSigner?

Reply:
Yeah..
I better update the `aws_request_signer` tests as well then. (separate PR)
Good catch. 👍